### PR TITLE
Initial draft of the TC donut

### PR DESCRIPTION
**Pull request by dogonthehorizon**
Mostly re-formatted from our internal docs.

### CODE REVIEW

File: project-roles/trusted-committer.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+they consistently receive feedback, features, and bug-fixes from a contributor.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Problem
+
+A project maintainer receives several contributions from a single contributor
+but is unclear on how to properly recognize these contributions.
+
+# Forces
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Another force could be that it's hard to be a 1-man show?  Or even a small team.

File: project-roles/trusted-committer.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+  - Over the lifecycle of a project the focus of the maintainers may shift away
+  to accomodate changing business priorities
+  - Potential trusted committers seek visible artifacts of their contributions
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: We shouldn't use the language of the solution in setting up the problem or its forces.

File: project-roles/trusted-committer.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+are going well, thank the TC for their continued effort in making the project
+successful and set a new date to check-in.
+
+## Sunsetting a Trusted Committer
+
+There comes a time when removal of a Trusted Committer is necessary, such as:
+
+* No longer willing to take part
+* No longer able to perform their duties
+* No longer employed by Nike
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Remove the name of Nike

Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Elsewhere in the document as well (except for known instances).

File: project-roles/trusted-committer.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+As a project matures, contributors can become less familiar with key aspects
+of a project.  Trusted Committers fill in these gaps.  This ensures that all
+aspects of the project are better served over time.
+
+A healthy set of Trusted Committers ensures that if project maintainers move on
+there is a plan for responsible stewardship at Nike.
+
+# Known Instances
+
+This has been tried and proven successful at Nike.
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Should give a nod and add PayPal here as well.

File: project-roles/trusted-committer.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+* No longer willing to take part
+* No longer able to perform their duties
+* No longer employed by Nike
+
+In each of the above cases a plan for removing access to project resources
+should agreed upon by both parties. This includes their entry in a project’s
+**Trusted Committer** section.
+
+After access is removed it is courteous to thank the TC for their participation
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: If you want to you an link to the `Praise Participants` article in this repo.  It is in the patterns repo.

File: project-roles/trusted-committer.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+## For Contributors
+
+Achieving Trusted Committer status for a project is a sign that the contributor
+has demonstrated an improvement to a community project. Recognition for these
+efforts can be used during annual reviews with managers.
+
+## For Maintainers
+
+As a project matures, contributors can become less familiar with key aspects
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Did you mean to say `maintainers`?

File: project-roles/trusted-committer.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
+# Title
+
+Trusted Comitter
+
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Not the end of the world, but the current [template](https://github.com/dogonthehorizon/InnerSourcePatterns/blob/master/meta/pattern-template.md) includes a section called **Patlet** to be included here.  In addition to **Title**.  I've found it useful and like it.

File: project-roles/trusted-committer.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+# Context
+
+Many inner-source projects will find themselves in a situation where
+they consistently receive feedback, features, and bug-fixes from a contributor.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Problem
+
+A project maintainer receives several contributions from a single contributor
```
Comment (rrrutledge) [inline project-roles/trusted-committer.md]: Include points about scaling the project with more support in a single point in time and also across time as maintainers shift.

File: project-roles/trusted-committer.md (hunk 1 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Thanks.

File: project-roles/trusted-committer.md (hunk 2 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: OK!

File: project-roles/trusted-committer.md (hunk 3 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Awesome.

File: project-roles/trusted-committer.md (hunk 4 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Thanks.

File: project-roles/trusted-committer.md (hunk 5 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Very nice!

File: project-roles/trusted-committer.md (hunk 6 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Looks good.

File: project-roles/trusted-committer.md (hunk 7 above)
Comment (rrrutledge) [reply project-roles/trusted-committer.md]: Awesome.

**Comment by dogonthehorizon** [comment]
@rrrutledge upstream PR here:
https://github.com/paypal/InnerSourcePatterns/pull/100

**Pull request merged**